quicklogic/common/toolchain_wrappers/symbiflow_generate_constraints

```diff
@@ -13,22 +13,14 @@ ARCH_DEF=$6
 
 if [[ "$DEVICE" =~ ^(qlf_k4n8_qlf_k4n8)$ ]];then
 	DEVICE_1="qlf_k4n8-qlf_k4n8_umc22"
-	PINMAPXML="pinmap_${PART}.xml"
+	PINMAPXML="pinmap_qlf_k4n8_umc22.xml"
 else
     DEVICE_1=${DEVICE}
 fi
 
-PINMAPCSV="pinmap_${PART}.csv"
```

Comment:
There is no default pinmap CSV file. Does it mean that a user has to always provide his own one? Maybe it would be good to use a default one (eg. the one from qlfpga_symbiflow_plugins) when none is provided.

Reply:
Comment from Lalit: "If user doesn’t have any idea about pin-mapping then it is better not to choose any pin-mapping rather we will not allow io placement and let PNR choose pin placement.
In that case, if -p option is not defined then do not pick any csv and in such case no pcf is allowed."